akka-docs/rst/project/migration-guide-persistence-experimental-2.3.x-2.4.x.rst

```diff
@@ -178,3 +178,19 @@ persistent actor on the sending side.
 
 Read more about at-least-once delivery in the :ref:`documentation for Scala <at-least-once-delivery>` and 
 :ref:`documentation for Java <at-least-once-delivery-java>`.  
+
+Default persistence plugins
+===========================
+Previously default ``akka.persistence.journal.plugin`` was set to the LevelDB journal ``akka.persistence.journal.leveldb``.
+Now persistence extension will instead use in-memory journal plugin ``akka.persistence.journal.inmem`` as a default.
+This change was needed to decouple persistence extension from the the LevelDB dependency.
```

Comment:
I agree with @patriknw, we cannot change the default back-end, only remove the default itself.